Unable to run this library

b47d470dde26f7df5a0b8b2b7c28480820bc24c8 seems to have made some changes to how the library is meant to be run, but the new method is unclear.

Running `./bin/cli.js` yields this error:
```
./bin/cli.js test.sketch 
internal/fs/utils.js:426
    throw new ERR_INVALID_ARG_TYPE(propName, ['string', 'Buffer', 'URL'], path);
    ^

TypeError [ERR_INVALID_ARG_TYPE]: The "path" argument must be one of type string, Buffer, or URL. Received type undefined
    at Object.openSync (fs.js:439:10)
    at Object.readFileSync (fs.js:349:35)
    at Function.fromFile (/path/node_modules/sketch-constructor/models/Sketch/Sketch.js:29:31)
    at convertSketchToSwiftUI (/path/switchToSwiftUI.js:9:12)
    at Object.<anonymous> (/path/bin/cli.js:22:1)
    at Module._compile (internal/modules/cjs/loader.js:774:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:785:10)
    at Module.load (internal/modules/cjs/loader.js:641:32)
    at Function.Module._load (internal/modules/cjs/loader.js:556:12)
    at Function.Module.runMain (internal/modules/cjs/loader.js:837:10)
```

Running `npm start` yields this error:

```
npm start test.sketch              

> sketch-to-swiftui@0.9.0 start $path
> node sketchToSwiftUI.js "test.sketch"

internal/modules/cjs/loader.js:626
    throw err;
    ^

Error: Cannot find module '/path/sketchToSwiftUI.js'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:623:15)
    at Function.Module._load (internal/modules/cjs/loader.js:527:27)
    at Function.Module.runMain (internal/modules/cjs/loader.js:837:10)
    at internal/main/run_main_module.js:17:11 {
  code: 'MODULE_NOT_FOUND',
  requireStack: []
}
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! sketch-to-swiftui@0.9.0 start: `node sketchToSwiftUI.js "test.sketch"`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the sketch-to-swiftui@0.9.0 start script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/kerrmarin/.npm/_logs/2020-04-08T09_08_58_581Z-debug.log
```

Running the file directly produces no output:
```
sketch-to-swiftUI|master ⇒ node switchToSwiftUI.js test.sketch
```

--- a/bin/cli.js
+++ b/bin/cli.js
@@ -1,6 +1,6 @@
 #!/usr/bin/env node
 
-const { convertSketchToSwiftUI } = require('../switchToSwiftUI')
+const { convertSketchToSwiftUI } = require('../sketchToSwiftUI')
 
 const argv = require('yargs')
     .scriptName('sketch-to-swiftUI')
@@ -19,4 +19,4 @@ const argv = require('yargs')
     .help('help')
     .argv
 
-convertSketchToSwiftUI(argv['_'][0], argv['o'])
+convertSketchToSwiftUI(argv['file'], argv['o'])

--- a/package.json
+++ b/package.json
@@ -5,7 +5,7 @@
   "bin": "./bin/cli.js",
   "main": "sketchToSwiftUI.js",
   "scripts": {
-    "start": "./bin/cli.js"
+    "convert": "./bin/cli.js"
   },
   "repository": "github:design-ops/sketch-to-swiftUI",
   "author": "Design Operations Collective",

--- a/package.json
+++ b/package.json
@@ -5,7 +5,7 @@
   "bin": "./bin/cli.js",
   "main": "sketchToSwiftUI.js",
   "scripts": {
-    "start": "node sketchToSwiftUI.js"
+    "start": "./bin/cli.js"
   },
   "repository": "github:design-ops/sketch-to-swiftUI",
   "author": "Design Operations Collective",

--- a/sketchToSwiftUI.js
+++ b/sketchToSwiftUI.js
@@ -5,7 +5,6 @@ const fs = require('fs')
 
 function convertSketchToSwiftUI(filename, outputFolder) {
     //console.log("Extracting from " + filename + " to " + outputFolder)
-
     Sketch.fromFile(filename)
         .then(sketch => {
             return processSketchFile(sketch, filename, outputFolder)
@@ -20,4 +19,4 @@ function convertSketchToSwiftUI(filename, outputFolder) {
         })
 }
 
-module.exports = {convertSketchToSwiftUI: convertSketchToSwiftUI}
+module.exports = {convertSketchToSwiftUI}

